src/client/components/immutable-list/immutable-list.tsx

```diff
@@ -128,17 +130,18 @@ export class ImmutableList<T> extends React.Component<ImmutableListProps<T>, Imm
   }
 
   render() {
-    const { items, getRows, label } = this.props;
+    const { items, getRows, label, toggleShowSuggestions } = this.props;
     const { editedIndex, pendingAddItem } = this.state;
 
     if (!items) return null;
-
+    let actions = [<button key='add' onClick={this.addItem.bind(this)}>Add item</button>];
+    if (toggleShowSuggestions) {
+      actions = [<button key='suggestions' onClick={toggleShowSuggestions.bind(this)}>Suggestions</button>].concat(actions);
+    }
     return <div className="immutable-list">
       <div className="list-title">
         <div className="label">{label}</div>
-        <div className="actions">
-          <button onClick={this.addItem.bind(this)}>Add item</button>
-        </div>
+        <div className="actions">{actions}</div>
```

Comment:
you can save yourself some typing and inventing keys by doing the ol:

```
{ toggleShowSuggestions  ? <button onClick={toggleShowSuggestions}>Suggestions</button> : null }
```

trick here :-p